app/assets/javascripts/snowcrash/modules/issues/table.js.coffee

```diff
@@ -108,17 +114,36 @@ class IssueTable
         data: data,
         dataType: 'json',
         success: (data) ->
+          $that.prop('checked', false)
           issue_id = $that.val()
 
           $($row.find('td')[that.tagColumnIndex]).replaceWith(data.tag_cell)
           $("#issues #issue_#{issue_id}").replaceWith(data.issue_link)
-          if $('input[type=checkbox]:checked').length == 0
-            $('.js-issue-actions').css('display', 'none')
+          if $('input[type=checkbox]:checked.js-multicheck').length == 0
+            that.resetToolbar()
 
         error: (foo,bar,foobar) ->
           $($row.find('td')[that.tagColumnIndex]).replaceWith("<td class='text-error'>Please try again</td>")
       }
 
+  onMergeSelected: (event) ->
```

Comment:
I think that the current /merging /merge URLs are not ideal in a RESTful / CRUD sense, maybe something like

GET /issues/merge/new
POST /issues/merge

Would be more resource-oriented?

Another idea, we don't need a discreet form field for each issue :id to be merged, we can pass them in a list `issues=1,32,124,532,1124` and split it server-side. That'd clean up the URL. In any case, I tend to agree that POST should be reserved to status-changing ops.